### PR TITLE
Add print_response_body

### PR DESCRIPTION
**Pull request by martindekov**
Adding print_response_body to the values.yml
and updaing the deployment.yml file to read
the value set to true by default. Also
bumping the charts

Signed-off-by: Martin Dekov <mdekov@vmware.com>

Deploying with charts would not give option to set output while reading
the kafka connector logs, this adds print_response_body value to values.yml
to enable the feature

## Description

Adding `print_response_body` to `values.yml` along with the field in `deployment.yml` to be red from the file. Bumping the charts in the process with `make charts`

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

Writing blog post which describes this and actually it is not feasible.

## How Has This Been Tested?

N/A

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Not sure if it is breaking?

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: chart/kafka-connector/values.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ replicas: 1
 gateway_url: http://gateway.openfaas:8080
 topics: faas-request
 print_response: true
+print_response_body: true
```
Comment (affix) [inline chart/kafka-connector/values.yaml]: current behaviour doesn't print the response body, consider setting the default value to false

Comment (alexellis) [reply chart/kafka-connector/values.yaml]: I would agree with this

Comment (martindekov) [reply chart/kafka-connector/values.yaml]: Thanks for the review Keiran, added commit which references that 👍

Comment (martindekov) [reply chart/kafka-connector/values.yaml]: Will squash when everything looks fine

**Comment by alexellis** [comment]
Please remove the changes of make charts. I prefer to do this after merge to prevent conflicts.

**Comment by martindekov** [comment]
Done 👍

**Comment by affix** [comment]
Looks good to me :)

**Comment by alexellis** [comment]
Thanks @affix & @martindekov

**Pull request merged**